README.md

```diff
@@ -807,15 +807,64 @@ When you have time, we invite you to take a look at:
  + [The dynamic router EIP](https://camel.apache.org/components/latest/eips/dynamicRouter-eip.html)
  + [The kamelet EIP](https://camel.apache.org/components/latest/eips/kamelet-eip.html)
 
-## Part 7 - Camel Quarkus and Kafka
+## Part 7 - Integration Example with Camel Quarkus
 Estimate time : 25 minutes
-@TODO: 25 MINUTES LEFT at this level
+
+### About this section
+In this part, we will discover how easy and fast it is to :
+* Stream data from and to Kafka
+* Write to Slack
+* Read / write to an SQL database with JPA
+* Use external APIs
+* Serialize Data
+* Create REST API
+
+
+We will also experience the joy of using [dev services](https://quarkus.io/guides/dev-services) in the development mode. No need to install any kafka server or database.
+
+---
+**🚀NOTE**
+
+This section requires having Docker installed in order to provition a Kafka and Postgresql servers. 
+
+In case, you don't have a proper Docker license:
+* Use H2 Databse instead of Postgresql
+* Ask us during the session to provide you a kafka instance in the Cloud.
+
+---
+
+### Introducing the example
+
+This example will illustrate a application that will receive and process coffee orders. The application is designed around Event Driven Architecture and uses Kafka to communicate with external systems. The orders are stored withing a database. The delivery team is notified on slack to go check the order. The delivery team can access information about orders to deliver using an API.
+
+
+* insert the orders within a Database
+* notify the delivery company using a specific API (This one will be provided during the session)
+
+The application is designed around Event Driven Architecture and uses Kafka to handle those messages:
+* The beer orders to be processed are in a kafka topic named `orders`
+* The notifications to the delivery company are in a kafka topic named `deliveries`
+
+The application that create orders and notify delivery teams are external to our ecosystem.
+
+![](images/CQ-workshop1.png)
+
 
 ## Bonus A - Camel Quarkus and the Kamelets
```

Comment:
At end of part-7 native mode: The last sentence "That's no big deal as we'll prefer to use the DEV and JVM mode for the rest of the workshop." is no more true now that we JVM + Native at the end ?

Reply:
Didn't understand this comment

Reply:
It's like in part-7, the last part of the workshop. We say, no big deal as we'll use the DEV mode from now on. Actually,the DEV mode has already been used before in other parts ? Now that this PR is re-ordering parts

Reply:
okay !! Understood now